src/ConsoleMode.ps1

```diff
@@ -0,0 +1,103 @@
+# Hack! https://gist.github.com/lzybkr/f2059cb2ee8d0c13c65ab933b75e998c
+
+if ($IsWindows -eq $false) {
```

Comment:
What was the initial impetus for this?  PowerShell sets the console mode to enable ANSI seqs. Or was there a version (5.0) that didn't do this?